python/lsst/daf/butler/registry/_dbAuth.py

```diff
@@ -88,16 +88,16 @@ def __init__(self, path=None, envVar=None, authList=None):
         except Exception as exc:
             raise DbAuthError(f"Unable to load DbAuth configuration file: {secretPath}.") from exc
 
-    def getAuth(self, drivername, username, host, port, database):
+    def getAuth(self, dialectname, username, host, port, database):
         """Retrieve a username and password for a database connection.
 
         This function matches elements from the database connection URL with
         glob-like URL patterns in a list of configuration dictionaries.
 
         Parameters
         ----------
-        drivername : `str`
-            Name of database backend driver from connection URL.
+        dialectname : `str`
+            Database dialect, f.e. sqlite, mysql, postgresql, oracle, or mssql.
```

Comment:
```suggestion
            Database dialect, for example sqlite, mysql, postgresql, oracle, or mssql.
```

Reply:
Expanded on other `f.e.`s in my docstrings too. Thanks!